pipeline/api/cloud.py

```diff
@@ -749,3 +751,13 @@ def _hash_file(self, file_path: str, block_size=2**20) -> str:
                     break
                 md5.update(data)
         return md5.hexdigest()
+
+    def get_runs(
+        self, limit: int = 20, skip: int = 0, created_at: str = "desc"
```

Comment:
`created_at_order` might be a better name, plus adding in some validation to check it's either `desc` or `asc`